types/twine-sugarcube/save.d.ts

```diff
@@ -33,6 +33,45 @@ export interface SaveObject {
     version?: any;
 }
 
+export interface SaveDetails {
+    /**
+     * A string representing how the save operation came about — i.e., what caused it.
+     */
+    type: 'autosave' | 'disk' | 'serialize' | 'slot';
+}
+
+type SaveHandler = (save: SaveObject, details: SaveDetails) => void;
+type LoadHandler = (save: SaveObject) => void;
+
+interface SaveEventAPI<HandlerType> {
+    /**
+     * Add new handler
+     * @param handler
+     */
+    add(handler: HandlerType): void;
+    /**
+     * Deletes all currently registered handlers.
+     */
+    clear(): void;
+    /**
+     * Deletes the specified handler.
```

Comment:
@ezsh The description has been updated in the repo to include return details.

Reply:
Thank, I've updated the description too.

Reply:
Yes, you added it in the `@returns`, but the description really should match the repo's.